sale_financial_risk/hooks.py

```diff
@@ -27,6 +27,7 @@ def create_commercial_partner_id_column(cr):
         SET commercial_partner_id = p.commercial_partner_id
         FROM res_partner p
         WHERE  p.id = sol.order_partner_id and
-                sol.commercial_partner_id <> p.commercial_partner_id;
+                (sol.commercial_partner_id <> p.commercial_partner_id
+                 or sol.commercial_partner_id is null);
```

Comment:
This expression cover all

```suggestion
                sol.commercial_partner_id IS DISTINCT FROM p.commercial_partner_id
```

Reply:
Changes done